redfishtool/Chassis.py

```diff
@@ -67,6 +68,7 @@ def displayOperations(self,rft):
         print("     setIndicatorLed  <state>  -- set the indicator LED.  <state>=redfish defined values: Off, Lit, Blinking")
         print("     Power                     -- get the full Power resource under a specified Chassis instance.")
         print("     Thermal                   -- get the full Thermal resource under a specified Chassis instance.")
+        print("     Sensors                   -- get all sensors.")
```

Comment:
Should also update README.md similarly to show this option in the "Systems Operations" section.